website/src/pages/create/assistant_reply.tsx

```diff
@@ -52,6 +56,7 @@ const AssistantReply = () => {
   }
 
   const task = tasks[0].task;
+  const endTask = tasks[tasks.length - 1];
```

Comment:
what is the advantage of having an array of tasks?

Reply:
This was originally based on how the demo `text-frontend` handled tasks.  After submitting the prototype was returning a potentially different task.

Now that it's changed to always return `task_done` after submission, I want to simplify all this in another PR that reduces code duplication.